motor_control/throttle/variable_control.ino

```diff
@@ -0,0 +1,25 @@
+#include "variable_control.h"
+
+VariableThrottle::VariableThrottle(const int &pin) : Throttle(pin) {}
+
+void VariableThrottle::compute_motor_value()
+{
+    /*
+   * Reads the throttle input from analog signal,
+   * Returns 0~100
+   */
+    int throttleReading = get_raw_value();
```

Comment:
Is there a reason we don't directly just use `m_raw_value`?

Reply:
@rgimel just wanted to ping this comment again real quick. Since you made `m_raw_value` a protected data member, you can access him directly here instead of by using `get_raw_value()`. You also use this getter function in one other place, so please make sure you just swap those out with `m_raw_value`!

Reply:
The getter functions are more for the consumers of this class to access those values.

Reply:
Oh lol it looks like I forgot to save before pushing to GitHub, I'll reupload now, sorry about that